infra-templates/ipsec/15/docker-compose.yml.tpl

```diff
@@ -12,7 +12,7 @@ services:
       - 500:500/udp
       - 4500:4500/udp
     labels:
-      io.rancher.sidekicks: router
+      io.rancher.sidekicks: router,hc
```

Comment:
let's not abbreviate the service. let's make it a name that people understand

Reply:
sure, will change it to `healthcheck`